Source/WebCore/page/writing-tools/WritingToolsController.h

```diff
@@ -116,6 +116,17 @@ class WritingToolsController final {
         using Value = CompositionState;
     };
 
+    class WritingToolsEditingScope {
```

Comment:
I generally dislike these generally hard-to-reason-about RAII classes, but I guess there isn't a better alternative

Reply:
maybe this can be renamed to `EditingScope`, since it's within the `WritingToolsController` class anyways?

Reply:
Good point.

Reply:
Agree with the name suggestion.

Reply:
Done.